src/renderer/components/+apps-releases/get-release-secret/get-release-secret.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import type { HelmRelease } from "../../../../common/k8s-api/endpoints/helm-releases.api";
+import type { SecretsStore } from "../../+config-secrets/secrets.store";
+
+interface Dependencies {
+  secretsStore: SecretsStore;
+}
+
+export const getReleaseSecret =
+  ({ secretsStore }: Dependencies) =>
+    (release: HelmRelease) => {
+      return secretsStore
```

Comment:
[nit] return can be left out for consistency

Reply:
Nice catch. Resolved